h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -86,6 +86,10 @@ protected ModelBuilder(P parms, Job<M> job) {
   protected ModelBuilder(P parms, boolean startup_once) { this(parms,startup_once,"hex.schemas."); }
   protected ModelBuilder(P parms, boolean startup_once, String externalSchemaDirectory ) {
     String base = getClass().getSimpleName().toLowerCase();
+    if (H2O.ARGS.features_level.compareTo(builderVisibility()) > 0) {
```

Comment:
This will not let you use the experimental algorithm in a stable algorithm, it will most likely cause a bad failure of the stable algo.

I would not modify model builder at all. The unstable algos just shouldn't be registered in RegisterAlgos.

Reply:
That was my first though. But then those are still listen in e.g. Flow and anything that fetches the list.

Reply:
IMHO, Flow filters out anything that is not stable by default. That is why you don't see Isolation Forest in Flow.